### ISSUE TITLE
lights wont turn on if set to 100%

### ISSUE THREAD

**Issue by domi-a** [andrei-tatar/node-red-contrib-nora#92]
reproduce:
- node has checked: Turn on when brightness/color changes
- turn light off (manually, by input or assistant)

- assistant: turn light to 100 percent
brightness is set to 100 but **payload.on** is still **false**

- assistant: turn light to 99 percent will actually work
**payload.on** will be **true**

**Comment by Barabba11** [comment]
some lights support maximum brightness 254, can this concept somehow influence your result? is 100% translated to 255?

**Comment by domi-a** [comment]
> some lights support maximum brightness 254, can this concept somehow influence your result? is 100% translated to 255?

no. it is the problem with the "on" property of payload like i wrote
